tag.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
```

Comment:
Is there a reason to not have this as `#!/usr/bin/env bash`

Maybe I'm the only one running this on MacOSX, but by default, MacOS's bash at /bin/bash is 3.2.57 - released in 2007, and Apple clearly does not intend to upgrade the built-in version. I've often run into problems where a bash construct created sometime in the last 13 years has been used in a script, and the built-in /bin/bash chokes on it.

With `#!/usr/bin/env bash`, it will pick up the first bash from `$PATH`, which is easy to make point to an up-to-date bash with something like homebrew.

Reply:
Yeah, that sounds good. More habit that I write `#!/bin/bash`.  I'll switch.